exchanges/okx/okx_test.go

```diff
@@ -0,0 +1,3129 @@
+		t.Error("Okx GetMaximumBuySellAmountOROpenAmount() error", err)
+	}
+}
+
+func TestGetMaximumAvailableTradableAmount(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	// Operation is not supported under the current account mode
+	if _, err := ok.GetMaximumAvailableTradableAmount(context.Background(), "BTC-USDT", "BTC", "cross", true, 123); err != nil && !strings.Contains(err.Error(), "51010") {
+		t.Error("Okx GetMaximumAvailableTradableAmount() error", err)
+	}
+}
+
+func TestIncreaseDecreaseMargin(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.IncreaseDecreaseMargin(context.Background(), IncreaseDecreaseMarginInput{
+		InstrumentID: "BTC-USDT",
+		PositionSide: "long",
+		Type:         "add",
+		Amount:       1000,
+		Currency:     "USD",
+	}); err != nil && !strings.Contains(err.Error(), "Unsupported operation") {
+		t.Error("Okx IncreaseDecreaseMargin() error", err)
+	}
+}
+
+func TestGetLeverage(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.GetLeverage(context.Background(), "BTC-USDT", "cross"); err != nil {
+		t.Error("Okx GetLeverage() error", err)
+	}
+}
+
+func TestGetMaximumLoanOfInstrument(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	// Operation is not supported under the current account mode
+	if _, err := ok.GetMaximumLoanOfInstrument(context.Background(), "ZRX-BTC", "isolated", "ZRX"); err != nil && !strings.Contains(err.Error(), "51010") {
+		t.Error("Okx GetMaximumLoanOfInstrument() error", err)
+	}
+}
+
+func TestGetFeeRate(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.GetTradeFee(context.Background(), "SPOT", "", ""); err != nil {
+		t.Error("Okx GetTradeFeeRate() error", err)
+	}
+}
+
+func TestGetInterestAccruedData(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.GetInterestAccruedData(context.Background(), 0, 1, "", "", "", time.Time{}, time.Time{}); err != nil {
+		t.Error("Okx GetInterestAccruedData() error", err)
+	}
+}
+
+func TestGetInterestRate(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.GetInterestRate(context.Background(), ""); err != nil {
+		t.Error("Okx GetInterestRate() error", err)
+	}
+}
+
+func TestSetGreeks(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.SetGreeks(context.Background(), "PA"); err != nil {
+		t.Error("Okx SetGeeks() error", err)
+	}
+}
+
+func TestIsolatedMarginTradingSettings(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.IsolatedMarginTradingSettings(context.Background(), IsolatedMode{
+		IsoMode:        "autonomy",
+		InstrumentType: "MARGIN",
+	}); err != nil && !strings.Contains(err.Error(), "51010") { // Operation is not supported under the current account mode
```

Comment:
```suggestion
	}); err != nil && !strings.Contains(err.Error(), "51010") { mode
```
Remove all comments of `// Operation is not supported under the current account `, we want to support more than your current API credential set